File: rating_service/src/main/java/huy/project/rating_service/core/domain/dto/request/BaseParams.java
package huy.project.rating_service.core.domain.dto.request;

import jakarta.validation.constraints.Min;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.experimental.SuperBuilder;

@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
@SuperBuilder
public class BaseParams {
    @Min(value = 0, message = "Trang phải lớn hơn hoặc bằng 0")
    private Integer page;

    @Min(value = 1, message = "Kích thước trang phải lớn hơn 0")
    private Integer pageSize;

    private String sortBy;
    private String sortType;
}

File: location_service/LocationService.Core/Domain/Dto/Request/BaseParams.cs
namespace LocationService.Core.Domain.Dto.Request
{
    public class BaseParams
    {
        public int Page { get; set; }
        public int PageSize { get; set; }
        public string? SortBy { get; set; }
        public string? SortOrder { get; set; }
    }
}